integration-tests/.env

```diff
@@ -7,7 +7,7 @@ POSTGRES_PORT=5432
 POSTGRES_DB=registry
 HIVE_ENCRYPTION_SECRET=wowverysecuremuchsecret
 HIVE_EMAIL_FROM=no-reply@graphql-hive.com
-HIVE_APP_BASE_URL=http://localhost:8080
+HIVE_APP_BASE_URL=http://0.0.0.0:8080
```

Comment:
This is should either be localhost or 127.0.0.1.

Reply:
https://github.com/kamilkisiela/graphql-hive/pull/1317
it was `0.0.0.0` before